grails-app/views/common/_hotjar.gsp

```diff
@@ -11,5 +11,11 @@
         r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
         a.appendChild(r);
     })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
+
+    // Hotjar user attributes
+    // https://help.hotjar.com/hc/en-us/articles/4402892526487-What-are-User-Attributes-
+    // https://help.hotjar.com/hc/en-us/articles/360038394053-How-to-Set-Up-User-Attributes
+    var userId = '${session?.user?.id}' || null;
+    window.hj('identify', userId, {});
```

Comment:
@jmiranda The `{}` is a custom user attribute config. Do we need to track now something else than only the user id? Examples can be seen here: https://help.hotjar.com/hc/en-us/articles/360038394053-How-to-Set-Up-User-Attributes

Reply:
At the moment I don't think we know what we'll need. Let's take it one step at a time though. Once we get the username set we'll be able to do some testing to see what else would be useful.